docs/monitoring.md

```diff
@@ -635,6 +635,7 @@ set of sinks to which metrics are reported. The following instances are currentl
 * `driver`: The Spark driver process (the process in which your SparkContext is created).
 * `shuffleService`: The Spark shuffle service.
 * `applicationMaster`: The Spark ApplicationMaster when running on YARN.
+* `mesos_cluster`: The Spark cluster scheduler when running on Mesos.
```

Comment:
Are there more like this? maybe there are other resource-manager-specific values we haven't added yet

Reply:
@srowen  Thanks !  I have searched the whole spark project, and found no other resource-manager-specific instances.